crates/ruff_linter/src/rules/pyflakes/rules/f_string_missing_placeholders.rs

```diff
@@ -47,16 +47,15 @@ impl AlwaysFixableViolation for FStringMissingPlaceholders {
 
 /// F541
 pub(crate) fn f_string_missing_placeholders(checker: &mut Checker, expr: &ast::ExprFString) {
-    if expr.value.f_strings().any(|f_string| {
-        f_string
+    for f_string in expr.value.f_strings() {
+        if f_string
             .elements
             .iter()
             .any(ast::FStringElement::is_expression)
-    }) {
-        return;
-    }
+        {
+            return;
```

Comment:
I think we should move on to the next f-string instead of skipping the remaining ones.

```suggestion
            continue;
```

Otherwise any f-string without a placeholder which follows the one with a placeholder will not be checked:

```py
f"no placeholder" f"has {placeholder}" f"no placeholder"
```

Can you also add this as a test case?